File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { RepresentOfficialsSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await RepresentOfficialsSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- RepresentOfficials SDK exists test

local sdk = require("represent-officials_sdk")

describe("RepresentOfficialsSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
